content/en/database_monitoring/architecture.md

```diff
@@ -0,0 +1,99 @@
+
+* [System metrics collected on Postgres][2]
+* [System metrics collected on MySQL][3]
+
+
+For self-hosted setups, you should install the Agent directly onto the database host so that you have full visibility into the health of your system running the database process.
+
+You need to grant the Agent read-only access to your database, as well as set up the integration configurations. The Agent needs to be able to login as a user to be able to run read-only queries on your database.
+
+Instructions for setting up Database Monitoring with a self-hosted provider:
+
+* [Postgres][4]
+* [MySQL][5]
+
+
+#### Cloud-managed
+
+If your setup is cloud-managed (with providers such as [AWS RDS][6] or Aurora, Google Cloud SQL, or Azure), the Agent must be installed remotely on a separate host and configured to connect to each managed instance.
+
+Database monitoring collects system metrics such as CPU, memory, disk usage, logs, and related telemetry directly from the cloud provider using our integration.
+
+{{< img src="database_monitoring/dbm_architecture_cloud-hosted.png" alt="Some really good alt text" style="width:70%;">}}
+
+You can install the Agent on any cloud VM (for example, EC2) as long as it can connect to your database instance(s).
+
+If you are not already running your own Kubernetes cluster, Datadog recommends using your cloud provider’s orchestration tools. For example, you can use [AWS ECS][7] to host the Datadog Agent, as [the Agent already exists as a Docker container][8].
+
+##### Kubernetes
+
+If you’re already running your apps on [Kubernetes][9], Datadog recommends using the [Datadog Cluster Agent with Database Monitoring][10], as this allows you to run [cluster checks][11] across your pods.
+
+{{< img src="database_monitoring/dbm_architecture_clusters.png" alt="Some really good alt text" style="width:70%;">}}
+
+Using the [Cluster Agent][12] is preferred because it distributes the Database instances across a pool of agents for you. This ensures that only one instance of each check runs as opposed to each node-based Agent pod running this corresponding check. The Cluster Agent holds the configurations and dynamically dispatches them to node-based Agents. The Agents on each node connect to the Cluster Agent every 10 seconds and retrieve the configurations to run. If an Agent stops reporting, the Cluster Agent removes it from the active pool and dispatches the configurations to other Agents. This ensures one (and only one) instance always runs even as nodes are added and removed from the cluster. This becomes important when you have a large number of Database instances so that the Cluster Agent can spread out the cluster checks across the different nodes.
+
+
+
+##### Aurora
+
+If you’re using [Aurora][13], the Agent needs to be connected to the individual Aurora instance (not the cluster endpoint) because the Agent must connect directly to the host being monitored.
```

Comment:
```suggestion
If you are using [Aurora][13], the Agent needs to be connected to the individual Aurora instance (not the cluster endpoint) because the Agent must connect directly to the host being monitored.
```